docs/how-to-guides/deployment-options.md

```diff
@@ -0,0 +1,160 @@
+
+```
+az deployment sub create --location <Location> --template-file FhirConverter-SingleAzureDeploy.bicep --parameters deployApplicationInsights=false
+```
+
+Note: for any time when the service is running while Application Insights is disabled, you will not have access to metrics and request logs that were captured during that time.
+
+**2. Security settings for the API endpoints are disabled.**
+
+It is **strongly** recommended to enable security for your FHIR Converter service. To enable security settings for the APIs, include `--parameters securityEnabled=true` and additional relevant security arguments in the deployment command (see more details in the [Configuration Settings](configuration-settings.md) document):
+
+```
+az deployment sub create --location <Location> --template-file FhirConverter-SingleAzureDeploy.bicep --parameters securityEnabled=true securityAuthenticationAudiences=<Audiences> securityAuthenticationAuthority=<Authority>
+```
+
+**3. Template store integration is disabled**
+
+When template store integration is disabled, a storage account will not be provisioned with the deployment and the FHIR Converter service will use the provided default templates. To use custom templates, template store integration must be enabled so that custom templates can be stored in the deployed storage account; to deploy a new storage account, include the `--parameters templateStoreEnabled=true` argument in the deployment command:
+```
+az deployment sub create --location <Location> --template-file FhirConverter-SingleAzureDeploy.bicep --parameters deployTemplateStore=true
+
+```
+
+When no custom storage account and storage container names are provided, default names will be generated using the serviceName you provide. If you want to customize the storage account and storage container names, include the --parameters templateStorageAccountName=<StorageAccountName> templateStorageAccountContainerName=<StorageContainerName> arguments in the deployment command:
+
+```
+az deployment sub create --location <Location> --template-file FhirConverter-SingleAzureDeploy.bicep --parameters deployTemplateStore=true templateStorageAccountName=<StorageAccountName> templateStorageAccountContainerName=<StorageContainerName>
+```
+
+Alternatively, if you want to enable template store integration but already have a storage account and storage container that you want to use, include the --parameters storageAccountName=<StorageAccountName> storageContainerName=<StorageContainerName> arguments in the deployment command, leaving the deployTemplateStore parameter as false:
+
+```
+az deployment sub create --location <Location> --template-file FhirConverter-SingleAzureDeploy.bicep --parameters storageAccountName=<StorageAccountName> storageContainerName=<StorageContainerName>
+```
```

Comment:
Couple suggestions to simplify this:
* List in bullets or tabular format, the parameters supported. 
   * We can specify for each, the default value and the configurable values.
 * We should just give the default quickstart versions here. If we list the parameters as suggested above, it will be clear what parameters to use with which value if they want to customize and avoids us having to show examples for each.
 * In each of the configuration settings doc, we should specify the deployment command with the expected parameter in there.